imports/plugins/included/surcharges/server/no-meteor/startup.js

```diff
@@ -23,6 +23,12 @@ export default function startup(context) {
     const cartSurcharges = [];
 
     // Merge surcharges from each shipping group
+    if (cart.accountId && !context.account) {
+      const { Accounts } = context.collections;
+      const account = await Accounts.findOne({ _id: cart.accountId });
+      context.account = account;
```

Comment:
The cart account may not be the same as the request account, and may have different permissions. Also, by mutating the “base” context, this value might stick around longer than you’re expecting and be used in subsequent event handler calls.

If you instead do `contextWithAccount = { ...context, account, accountId: cart.accountId };` and then pass through `contextWithAccount` instead of `context` below, that's safe, though potentially not the best fix in the long run.

Reply:
Should be fixed in bdc57ed50